ERCS/erc-7566.md

```diff
@@ -0,0 +1,193 @@
+---
+eip: 7566
+title: Multiplayer Onchain Game
+description: A solution that enables multiple players to play onchain game.
+author: Rickey (@HelloRickey)
+discussions-to: https://ethereum-magicians.org/t/erc-proposal-multiplayer-onchain-game/16796
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-11-28
+---
+
+## Abstract
+
+This proposal proposes a multiplayer onchain game (MOG) standard interface based on smart contracts, using room to match and group players, and using message to process actions between players. This allows one smart contract to handle multiple players playing games on the chain, preventing centralized servers from affecting the fairness of the game.
```

Comment:
> using room
> using message

You should explain a bit more about what "room" and "message" are here. Are they contracts, abstract concepts, or something else? If they are contracts or interfaces, you could do something like `Room` or `Message` to indicate that they are snippets of code.